src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
```

Comment:
You have set the width of the .header class to 100%. This is not necessary because block-level elements, like a div with a class of .header, by default take up the full width of their parent. Unless you have a specific reason for including this, you can safely remove it.